prow/plugins/lgtm/lgtm.go

```diff
@@ -57,7 +57,7 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 		Description: "Adds or removes the 'lgtm' label which is typically used to gate merging.",
 		Featured:    true,
 		WhoCanUse:   "Collaborators on the repository. '/lgtm cancel' can be used additionally by the PR author.",
-		Examples:    []string{"/lgtm", "/lgtm cancel", "toggle the Review button to 'Approve' or 'Request Changes' in the github GUI"},
+		Examples:    []string{"/lgtm", "/lgtm cancel", "<a href=\"https://help.github.com/articles/about-pull-request-reviews/\">'Approve' or 'Request Changes'</a>"},
```

Comment:
worth noting I edited these changes into the plugin-help.js data by hand because run-local doesn't support local data for this. I haven't looked super thoroughly to verify that html is supported here, but it looks like it should work tentatively, as long as we're not stripping it out somewhere in the API, the frontend supports it.